### PR TITLE
Add keymap v2 and v3 for soda/mango

### PR DESCRIPTION
**Pull request by gezhaoyou**
## Description
Add keymap v2 and v3 for soda/mango

## QMK Pull Request 
https://github.com/qmk/qmk_firmware/pull/21100#event-9438160621

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/soda/mango/mango.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
+    "name": "MF17",
+    "vendorId": "0x1A54",
```
Comment (Xelus22) [inline v3/soda/mango/mango.json:3]: wrong VID

**Review by Xelus22 — CHANGES_REQUESTED** [review]
Please delete v2 definition as this is not a legacy keyboard (src/ folder json)

Are you sure this is correct?
image

File: v3/soda/mango/mango.json (hunk 1 above)
Comment (gezhaoyou) [reply v3/soda/mango/mango.json:3]: I have  update v3 vid and rm src v2 json

Comment (gezhaoyou) [reply v3/soda/mango/mango.json:3]: My keyboard is already in mass production, I need this vid, can't change it. Please let me pass

**Comment by gezhaoyou** [comment]
> Please delete v2 definition as this is not a legacy keyboard (src/ folder json)
> 
> Are you sure this is correct? image

This is not a problem and has no impact on the operation of my keyboard.

**Pull request closed without merging**